lib/pbench/server/api/__init__.py

```diff
@@ -133,6 +134,13 @@ def register_endpoints(api: Api, app: Flask, config: PbenchServerConfig):
         endpoint="key",
         resource_class_args=(config,),
     )
+    api.add_resource(
+        Visualize,
+        f"{base_uri}/visualize/<string:dataset>",
+        f"{base_uri}/visualize/<string:dataset>/",
```

Comment:
Again, I don't think we want the second trailing slash form here. We use that when we have a second parameter and want to support calls with that omitted (as with the `<path>` on contents and inventory).